content/_code-samples/normalize-currency-codes/js/normalize-currency-code.js

```diff
@@ -0,0 +1,65 @@
+////////////////////////////////////////////////////////////////////////////////
+// Normalize Currency Codes
+// Original authors: Ali (XUMM), nixer89
+// Upstream version: https://github.com/nixer89/xrpl-services-frontend/blob/master/src/app/utils/normalizers.ts
+////////////////////////////////////////////////////////////////////////////////
+// Convert from a string from either the XRP Ledger's "standard" 3-character
+// format, or several possible decodings of the "non-standard" 40-character
+// hexadecimal format into a string for humans to read.
+//
+// This code is intended for use with Node.js.
+////////////////////////////////////////////////////////////////////////////////
+
+function convertDemurrageToUTF8(demurrageCode) {
```

Comment:
What does `demurrage` mean? (Maybe there's a more intuitive word for it?)

Reply:
https://en.wikipedia.org/wiki/Demurrage_(currency)
https://xrpl.org/demurrage.html

I'm not aware of any other words for demurrage, but I think it would be fine to add one or both of the above links for anyone reading the code to reference.

Reply:
The latter link already is in the code (later on)